CHANGELOG.md

```diff
@@ -1 +1,2 @@
 - Add sharp NPM module to Cloud Functions when using Next.js Image Optimization
+- Adds user-defined env vars into the functions emulator (#5330).
```

Comment:
the most trivial conceivable nit--this sentence ends with a period but the one above it doesn't?

Reply:
🤣   I'm a big fan of our changelog's ending in periods, so I added one to the first entry!